const-c.inc

```diff
@@ -0,0 +1,55 @@
+#define PERL_constant_NOTFOUND	1
```

Comment:
You are responsible for every line of code, because the PR states, that Shahrizal is an author :) That's why it is better to avoid all autogenerators. In any case, I'll will not accept the PR, if you'll not be able to explain the need of any line of code.